app/views/projects/new.html.erb

```diff
@@ -0,0 +1,5 @@
+<%- model_class = Project -%>
```

Comment:
I don't think this needs the hyphen when opening the tag, so `<%` instead of `<%-`. Not sure though, just something I vaguely remember.